apps/ledger-live-mobile/src/components/ReadOnlyFabActions.tsx

```diff
@@ -6,17 +6,34 @@ import { useNavigation } from "@react-navigation/native";
 import Button from "./wrappedUi/Button";
 
 import { ScreenName } from "../const";
+import useCurrency from "../helpers/useCurrency";
+import { useCurrentRouteName } from "../helpers/routeHooks";
+import { track } from "../analytics";
 
 const iconBuy = Icons.PlusMedium;
 const iconReceive = Icons.ArrowBottomMedium;
 
 function ReadOnlyFabActions() {
   const { t } = useTranslation();
   const { navigate } = useNavigation();
+  const currentRoute = useCurrentRouteName();
+  const currency = useCurrency().name;
 
-  const handleOnPress = useCallback(() => {
-    navigate(ScreenName.BuyDeviceScreen);
-  }, [navigate]);
+  const buyDevice = useCallback(() => navigate(ScreenName.BuyDeviceScreen), [
+    navigate,
+  ]);
+
+  const handleOnPress = useCallback(
```

Comment:
useCallback is beneficial to preserve reference,
here there are no reference preserved because it's getting `() =>` recreated inside the functions.

two possibilites:
- makes one useCallback for each button
- completely remove the useCallback and have this helper just leaving here.